src/core/Akka/Actor/ActorCell.cs

```diff
@@ -452,7 +449,6 @@ protected void ClearActor(ActorBase actor)
                     }
                 }
 
-                ReleaseActor(actor);
```

Comment:
In the event of injected dependencies inside an actor, what's responsible for cleaning up those dependencies? The `scope.Dispose()` call earlier?

Reply:
Yes.

Reply:
Ok, and that's implemented by the DI container itself right?

Reply:
I can write tests to ensure it is.